src/Sentry.Unity/Integrations/AnrIntegration.cs

```diff
@@ -78,7 +78,10 @@ protected void Report()
         {
             var message = $"Application not responding for at least {DetectionTimeoutMs} ms.";
             Logger?.LogInfo("Detected an ANR event: {0}", message);
-            OnApplicationNotResponding?.Invoke(this, new ApplicationNotRespondingException(message));
+
+            var exception = new ApplicationNotRespondingException(message);
+            exception.SetSentryMechanism("ANR", handled: false);
```

Comment:
Could benefit some description. E.g: "doesn't crash the app"
I suggested that here: https://github.com/getsentry/sentry-unity/issues/1954